src/util.ts

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/env node
+
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+ * See LICENSE in the project root for license information.
+ */
+
+import * as fs from 'fs';
+import * as request from 'request';
+import * as rp from 'request-promise';
+import * as chalk from 'chalk';
+import * as appInsights from 'applicationinsights';
+
+let insight = appInsights.getClient('78cc7757-c7a2-4382-b801-bce73cf33d7a');
+let baseUri = 'https://verificationservice.osi.office.net/ova/addincheckingagent.svc/api/addincheck';
+let options = {
+  uri: baseUri,
+  method: 'POST',
+  headers: {
+    'Content-Type': 'application/xml'
+  },
+  resolveWithFullResponse: true
+};
+
+export async function validateManifest(manifest: string) : Promise<number> {
+  try {
+    console.log('Calling validation service...');
```

Comment:
Geoffrey and I spoke in person and decided that if we're taking the spinner away, we need to make this more descriptive (e.g. change to something like "Calling validation service. This could take a few moments...")